crates/gettext/src/lib.rs

```diff
@@ -1,259 +1,17 @@
 use fish_gettext_maps::CATALOGS;
 use once_cell::sync::Lazy;
-use std::{collections::HashSet, sync::Mutex};
+use std::sync::Mutex;
 
 type Catalog = &'static phf::Map<&'static str, &'static str>;
 
-pub struct SetLanguageLints<'a> {
-    pub duplicates: Vec<&'a str>,
```

Comment:
BTW I think it would be fine to put various unrelated commits into a PR to avoid trivial conflicts,
if it's easy to merge leading commits.
(Like the wchar prelude the module extraction ones though it's not really a relevant problem yet).

I'll merge the first commit from this one, since it seems ready (?)